### DISCUSSION TITLE
Do I need to see tray icon to know that program is working?

### DISCUSSION THREAD

**Discussion by JFMugen** [sandboxie-plus/Sandboxie#1339] [Q&A Feedback] [answered]
Heyyo, I recently discovered Sandboxie, and it is awesome! There is one thing that I don't understand tho, when I don't select "Start UI when sandboxed process started" option in the menu, I can't see the icon on the tray when I launch the program. 

Does that mean the program doesn't work (program can save outside of sandbox environment)? Or it just means the UI doesn't start, but the program still works as intended?

If it has to be open, I think "Start on tray" option would be great (actually it is a nice option even if it isn't "has to" imo :)

**Comment by DavidXanatos** [comment, marked as answer]
The tray ui is optional, without it the program's are still sandboxed.
If you have set to alter the window title of sandboxed programs you will see the markings in the title bar.
One thing though, the color border drawn around sandboxed programs is drawn by the UI component so without it that indication is not available.

**Reply by JFMugen** [reply]
Then let me turn this QA to Suggestion instead :) 

If it is not too much hassle to add, "Start on tray" option can be helpful. Definitely not a big deal but I find the color border pretty useful and I think it is bit annoying to manually close UI when I open a sandboxed program.
